rb/lib/selenium/webdriver/firefox/extension.rb

```diff
@@ -87,7 +87,10 @@ def read_id_from_manifest_json(directory)
           return unless File.exist?(manifest_path)
 
           manifest = JSON.parse(File.read(manifest_path))
-          [manifest['name'].delete(' '), manifest['version']].join('@')
+          id = if manifest.key?('application') && manifest['application'].key?('gecko')
```

Comment:
Couldn't you just write this as an if/else or a guard clause like on line 87? Just seems a bit weird doing this conditional assignment for essentially an if/else.

Reply:
@luke-hill Thanks for advice